astropy/coordinates/transformations.py

```diff
@@ -675,6 +675,38 @@ def deco(func):
             return func
         return deco
 
+    @contextmanager
+    def impose_finite_difference_dt(self, dt):
+        """
+        Context manager to impose a finite-difference time step on all applicable transformations
+
+        For each transformation in this transformation graph that has the attribute
+        ``finite_difference_dt``, that attribute is set to the provided value.  The only standard
+        transformation with this attribute is
+        `~astropy.coordinates.transformations.FunctionTransformWithFiniteDifference`.
+
+        Parameters
+        ----------
+        dt : `~astropy.units.Quantity` or callable
+            If a quantity, this is the size of the differential used to do the finite difference.
+            If a callable, should accept ``(fromcoord, toframe)`` and return the ``dt`` value.
+        """
+        key = 'finite_difference_dt'
+        saved_settings = []
+
+        try:
+            for from_frame in self._graph:
```

Comment:
since `self._graph` is just a dict, you can iterate over `.values()`
```
for to_frames in self._graph.values():
    for transform in to_frames.values():
```

Reply:
Ah, good point!

Reply:
Done